app/presenters/queries/version_presenter.rb

```diff
@@ -0,0 +1,45 @@
+module Queries
```

Comment:
We typically use presenters for making transformations before API requests, which isn't a major issue.

However, I'm just wondering how frequently this gets used, my suspicion is that it's only used to eventually pass data into the AuditTrailEntry component. I wonder if we could do away with this class and do the presenting in the AuditTrailEntry component which doesn't have much behaviour on it at the moment. May not work but might be worth a try 🤔